lisa/tests/staging/sched_android.py

```diff
@@ -0,0 +1,204 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2019, Arm Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import os.path
+import abc
+
+from lisa.wlgen.rta import RTA, Periodic
+from lisa.tests.base import ResultBundle, CannotCreateError, RTATestBundle
+from lisa.trace import Trace, FtraceCollector, FtraceConf, requires_events
+from lisa.target import Target
+from lisa.utils import ArtifactPath
+
+class SchedTuneTest(RTATestBundle, abc.ABC):
+    """
+    Abstract class for SchedTune testing
+
+    :param boost: The SchedTune `boost` level to apply the rtapp's cgroup
+    :type boost: int
+
+    :param prefer_idle: The SchedTune `prefer-idle` flag to apply to the rtapp's
+        cgroup
+    :type prefer_idle: bool
+
+    This class provides :meth:`get_cgroup` to create in the SchedTune
+    controller a `lisa_test` cgroup configured with the `boost` and
+    `prefer_idle` attributes.
+    """
+
+    boost = 0
```

Comment:
boost needs to be defined as an abstractmethod (since abstract attributes don't exist for `abc.ABC`, but it does not matter, it's the same at the end):
```
@abc.abstractmethod
def boost():
    """Your docstring"""
    pass
```

Otherwise, that class will not be `inspect.isabstract()` and exekall will try to create instances of it. Inheriting from `abc.ABC` does not make it abstract if all abstract methods are implemented (otherwise it would be impossible to be concrete, since there would be an ABC in your ancestors).